cli/cli.hs

```diff
@@ -108,7 +108,7 @@ commonOptions = CommonOptions
   <*> (switch $ long "trace"                <> help "Dump trace")
   <*> (option auto $ long "verb"            <> showDefault <> value 1 <> help "Append call trace: {1} failures {2} all")
   <*> (optional $ strOption $ long "root"   <> help "Path to  project root directory")
-  <*> (option auto $ long "assertion-type"  <> showDefault <> value Forge <> help "Assertions as per Forge or DSTest")
+  <*> (option auto $ long "assertion-type"  <> showDefault <> value Forge <> help "Assertions as per Forge, DSTest or Echidna")
```

Comment:
should be `[..] DSTest, or Echidna`, i.e. there is a comma missing. We should also explain what Echidna does in the html documentation someplace, once it's properly in.